site/docs/v1/tech/samlv2/zendesk.adoc

```diff
@@ -37,7 +37,9 @@ Next, we'll create a lambda to take FusionAuth role information and populate the
 [source,javascript]
 ----
 function populate(samlResponse, user, registration) {
-  samlResponse.assertion.attributes.role = registration.roles;
+  if (registration && registration.roles) {
+    samlResponse.assertion.attributes.role = registration.roles;
+  }
 }
```

Comment:
This outer function definition is missing from the screenshot.

Reply:
Doh!